### DISCUSSION TITLE
multi diffusion upscaler - install not activating

### DISCUSSION THREAD

**Discussion by galavera** [lllyasviel/stable-diffusion-webui-forge#492] [Q&A]
Hello everyone,

https://github.com/pkuliyi2015/multidiffusion-upscaler-for-automatic1111
I was using this extension on automatic1111 last couple of days with great results.

However when I use the in-built tile diffusion extension in forge it doesn't seem work correctly.

I tried installing this one and disabling the in-built extension on forge. But no matter what I do it doesn't let me activate this extension... any advice on how I can 'force' activate this would be appreciated. I enjoy using forge and would like not having to switch back and forth just to use this plugin.

Thanks!

**Comment by miro-ux** [comment]
Same issue. No solution yet...

**Reply by SvKora** [reply]
I have this problem too(

**Comment by gutsalovpavel** [comment]
the fact that the creator of Forge doesn't want to allow us tiled diffusion, means that there are some big issues with it

**Reply by miro-ux** [reply]
> the fact that the creator of Forge doesn't want to allow us tiled diffusion, means that there are some big issues with it

I wouldn't jump straight to bad faith man. Considering the complexity of how SD+Webui works there are probably a thousand other reasons it's not working apart from Forge creators don't want us to use it. Not to mention we all get it for absolutely free. If you don't have the skills to contribute towards fixing a bug, I'd advice at least showing gratitude and appreciation.

**Reply by gutsalovpavel** [reply]
sure sure, I don't have anything against forge creators, just want to say if they haven't do it yet, means it is a really complex task, so your comment is showing your thoughts more haha

**Comment by realmalikov** [comment]
https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/672

**Reply by miro-ux** [reply]
@realmalikov did this work for you? I was able to enable the extension but it errors and doesn't show up in the UI.

**Comment by THATDONFC** [comment]
I experienced the same issue. I'm wondering if it has something to do with the builtin multidiffusion extension

**Reply by miro-ux** [reply]
I tried disabling it by moving `/extensions-builtin/sd_forge_multidiffusion` out of that folder and it was disabled successfully but the manually installed muti diffuser still give me errors in terminal 😞

**Comment by gohan2091** [comment]
can't use this either
